docs/en/operation/StressBench.md

```diff
@@ -263,3 +263,154 @@ Here is a sample usage demo, where we have one master and three workers:
+single node mode. Use the `--cluster-limit` option to further specify how many job workers 
+should be used to run the benchmark job in parallel.
+
+Each individual RPC benchmark tests a specific RPC service offered by the master. A benchmark 
+may perform necessary preparations that simulate real use cases before the measurement 
+begins. Then the benchmark sends the RPC request(s) to the master once or repeatedly, depending on 
+the benchmark, and waits for the response(s). If the benchmark calls RPCs repeatedly, it will stop 
+when the specified measurement duration has been reached. Otherwise, it stops as soon as the 
+response is received.
+One successful RPC request results in one data point that records the round trip time of the RPC.
+
+For example, the `GetPinnedFileIdsBench` measures the RPC throughput of the `GetPinnedFileIds` RPC. 
+It creates test files in a temporary directory in Alluxio and pins them. Then it repeatedly calls 
+the RPC and waits for the pinned file list, records and outputs the data points and some statistics
+of the data.
+
+### Common Parameters
+
+These common parameters are available to all RPC benchmarks:
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>concurrency</td>
+        <td>2</td>
+        <td>The number of simulated clients on each node. Typically benchmarks use 1 thread for 1 
+        simulated client.</td>
+    </tr>
+    <tr>
+        <td>duration</td>
+        <td>5s</td>
+        <td>How long the benchmark should be recording measurements for.</td>
+    </tr>
+</table>
+
+### Available benchmarks
+
+These RPC benchmarks are currently available:
+
+- [`GetPinnedFileIdsBench`](#getpinnedfileidsbench)
+- [`RegisterWorkerBench`](#registerworkerbench)
+- [`WorkerHeartbeatBench`](#workerheartbeatbench)
+- [`StreamRegisterWorkerBench`](#streamregisterworkerbench)
+
+#### `GetPinnedFileIdsBench`
+
+For the `GetPinnedFileIds` RPC.
+
+This benchmark calls the RPC repeatedly until duration is reached.
+
+`GetPinnedFileIds` is periodically called by block workers on every heartbeat to poll the 
+pinned file list. It can be expensive if the pinned file list is huge. 
+
+Parameters:
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>num-files</td>
+        <td>100</td>
+        <td>Number of pinned files. Use this parameter to adjust the load incurred by the RPC. 
+            Higher numbers incur higher loads.</td>
+    </tr>
+    <tr>
+        <td>base-dir</td>
+        <td>/get-pin-list-bench-base</td>
+        <td>The temporary directory to contain the test files.</td>
+    </tr>
+</table>
+
+#### `RegisterWorkerBench` 
+
+For the `RegisterWorker` RPC and the `RequestRegisterLease` RPC.
```

Comment:
Simulates concurrent worker registration using the unary RPC implementation(default before Alluxio 2.7). Note that Alluxio 2.7 introduced the "register lease" for the master to perform registration flow control. If you enable that, the same flow control will be effective in this test too. You may use this command to simulate the pressure when the workers start at once at your scale.